lacework/provider.go

```diff
@@ -326,14 +326,14 @@ func verifyPrimaryAccount(account string, opts ...api.Option) (string, error) {
 		return account, err
 	}
 
-	orgInfo, err := lwApi.Account.GetOrganizationInfo()
+	orgInfo, err := lwApi.V2.OrganizationInfo.Get()
 	if err != nil {
 		return account, err
 	}
 
-	if orgInfo.OrgAccount {
+	if orgInfo.Data[0].OrgAccount {
```

Comment:
could we please check `len(orgInfo.Data) > 0` before accessing `Data[0]` - otherwise the provider will panic.

Reply:
Done. Adding this check in next commit.